src/System.Memory/src/System.Memory.csproj

```diff
@@ -27,12 +24,22 @@
     <Compile Include="System\SpanHelpers.Clear.cs" />
     <Compile Include="System\ThrowHelper.cs" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
```

Comment:
Is it possible to use CoreLib's Unsafe implementation instead? Maybe it would require making it public in CoreLib (but without a contract in CoreFX), which seems reasonable, as that's what we did for RuntimeThread as well.